components/ManualPreview/ManualPreview.jsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react'
+import Link from 'next/link'
+import cn from 'classnames'
+import Image from 'next/image'
+
+import styles from './ManualPreview.module.css'
+
+function ManualPreview({ title, subtitle, pageUrl, color, status, publishedDate, pattern, image }) {
+    return (
+        <Link
+            href={{
+                pathname: '/[[...pageUrl]]',
+                query: { pageUrl: [pageUrl] },
+            }}
+            className={styles.manual}
+        >
+            <div
+                className={styles.manualInner}
+                style={{ color, backgroundImage: pattern ? `url(${pattern})` : null }}
+            >
+                {title && <div className={styles.manualTitle}>{title}</div>}
+                {subtitle && <div className={styles.manualSubtitle}>{subtitle}</div>}
+
+                {status && (
+                    <div
+                        className={cn(styles.manualStatus, {
+                            [`${styles.manualStatusNew}`]: status === 'Новое',
```

Comment:
`const manualStatus = cn(...)`

Reply:
Why do we need to create a new variable? Usually `cn` is always applied in `className` and is not put into separate variables, including this project